docs/docs-beta/docs/dagster-plus/deployment/hybrid/agents/kubernetes.md

```diff
@@ -113,14 +138,315 @@ helm --namespace dagster-cloud upgrade agent \
 
-![Screenshot of agent health information](/img/placeholder.svg)
 
-### View logs
+:::tip
+For cloud based Kubernetes such as AWS EKS, AKS, or GCP, you typically don't need an image pull secret. The role used by Kubernetes will have permission to access the registry. You can skip this configuration.
+:::
+
+
+First create the secret. This step will vary based on the registry you use, but for DockerHub:
+
+```
+kubectl create secret docker-registry regCred \
+  --docker-server=DOCKER_REGISTRY_SERVER \
+  --docker-username=DOCKER_USER \
+  --docker-password=DOCKER_PASSWORD \
+  --docker-email=DOCKER_EMAIL
+```
+
+Use Helm to configure the agent with the secret:
+
+```yaml file=values.yaml
+# values.yaml
+imagePullSecrets: [regCred]
+```
 
 ```shell
-kubectl --namespace dagster-cloud logs -l deployment=agent
+helm --namespace dagster-cloud upgrade agent \
+    dagster-cloud/dagster-cloud-agent \
+    --values ./values.yaml
+```
+
+### Don't send Dagster+ stdout and stderr
+
+By default, Dagster+ will capture stdout and stderr and present them to users in the runs page. You may not want to send Dagster+ these logs, in which case you should update the compute logs setting.
+
+<Tabs>
+
+<TabItem value="s3" label="If you use AWS S3 at your organization">
+
+The compute logs for a run will be stored in your S3 bucket and a link will be presented to users in the Dagster run page.
+
+```yaml file=values.yaml
+# values.yaml
+computeLogs:
+  enabled: true
+  custom:
+    module: dagster_aws.s3.compute_log_manager
+    class: S3ComputeLogManager
+    config:
+      show_url_only: true
+      bucket: your-compute-log-storage-bucket
+      region: your-bucket-region
+```
+
+```shell
+helm --namespace dagster-cloud upgrade agent \
+    dagster-cloud/dagster-cloud-agent \
+    --values ./values.yaml
+```
+
+</TabItem>
+<TabItem value="disable" label="You don't use AWS S3">
+
+You can turn off compute logs altogether which will prevent Dagster+ from storing stdout and stderr. This setting will prevent users from accessing these logs.
+
+```yaml file=values.yaml
+# values.yaml
+computeLogs:
+  enabled: false
+```
+
+```shell
+helm --namespace dagster-cloud upgrade agent \
+    dagster-cloud/dagster-cloud-agent \
+    --values ./values.yaml
+```
+
+It's also possible to write your own compute log manager.
+
+</TabItem>
+</Tabs>
+
+### Make secrets available to your code
+
+You can make secrets available [through the Dagster+ web interface](/dagster-plus/deployment/environment-variables) or through Kubernetes. Configuring secrets through Kubernetes has the benefit that Dagster+ never stores or accesses the secrets. First, create the Kubernetes secret:
+
+```bash
+kubectl create secret generic database-password \
```

Comment:
you could consider making the secret name more distinct from the env var name to prevent confusion